Rocket.Chat/API/Requests/UsersListRequest.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UsersListRequest.swift
+//  Rocket.Chat
+//
+//  Created by Artur Rymarz on 12.11.2017.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import SwiftyJSON
+
+typealias UsersListResult = APIResult<UsersListRequest>
+
+class UsersListRequest: APIRequest {
```

Comment:
Will change that, although it will be not possible to invite users who are not in local database. For servers with lots of channels it will be prettty common issue I think?